content/sklearn/concepts/kernel-ridge-regression/kernel-ridge-regression.md

```diff
@@ -0,0 +1,115 @@
+---
+Title: 'Kernel Ridge Regression'
+Description: 'Kernel ridge regression is a sophisticated linear regression model combined with L2 regularization and kernel trick to handle non-linearities that provide optimal solutions.'
+Subjects:
+  - 'Data Science'
+  - 'Machine Learning'
+Tags:
+  - 'Machine Learning'
+  - 'Python'
+  - 'Scikit-learn'
+CatalogContent:
+  - 'paths/data-science'
+  - 'paths/intermediate-machine-learning-skill-path'
```

Comment:
```suggestion
  - 'paths/intermediate-machine-learning-skill-path'
```
Add 'paths/intermediate-machine-learning-skill-path' in the [catalog-content.md file](documentation/catalog-content.md)

Reply:
Already in there